README.md

```
@@ -2,133 +2,250 @@
+npm run lint             # ESLint on scripts and tests
+npm run validate         # Company schema + kit/core config validation
+npm run validate:schema  # Schema validation only
 
-## Repository map
+# Run the parity audit — shows cross-tool feature gaps
+node scripts/parity-audit.js
+```
+
+Example parity audit output:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)